metadata_extract/resource_loader.py

```diff
@@ -0,0 +1,96 @@
+# pylint: disable=missing-module-docstring
+import json
+from importlib.resources import files
+from typing import Any, Optional
+
+
+class ResourceLoader:
+    """ Class for loading resource files in the metadata_extract.data directory,
+    which are applied to regular expressions in the metadata_extract.text module
+    and in the keyword list of the metadata.infopage module.
+
+    The application requires the following files used in this class to run:
+        - txt/info_page_keywords.json
+        - txt/stopwords.json
+        - txt/labels.json
+        - txt/doc_type_mapping_no_en.json
+    """
+    __info_page_keywords: list[str]
+    __stopwords: list[str]
+    __labels: dict[str, Any]
+    __doc_type_mapping: dict[str, str]
+
+    __lang_info_page_keywords: dict[str, list[str]]
+    __lang_stopwords: dict[str, list[str]]
+    __lang_labels: dict[str, Any]
+
+    @staticmethod
+    def load_info_page_keywords(selected_languages: Optional[list[str]] = None) -> None:
```

Comment:
As they are written now, the `load_` methods are called once, and build a dict out of the json file, then the `get_` methods build a list by flattening the dict.
Building the list every time `get_`is called is unnecessary. I suggest you rewrite the `load_` methods to directly build the lists. Keeping the dict internally is of no use.
Two other suggestions:

- the `load_` method could start with something like `if list is already filled, then return it` - that way if we mistakenly call it multiple times, the json file is still only read once.
- you could define a `load(seleced_languages)` method that call all 4 internal `load_...`, that could then be made private. That would make `Meteor.__init__()` more concise.